guides/prerequisites.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: page
+title: Prerequisites
+parent: Guides
+nav_order: 1
+---
+
+# Prerequisites
```

Comment:
Nitpick, should this be a subsection of Installation? I'm not entirely sure how these pages would appear (if at all) in a site that consumes this theme, but I was a bit surprised to see this as its own page.